practice-app/api/views.py

```diff
@@ -49,63 +49,154 @@ def post(self, *args, **kwargs):
         return Response(serializer.data, status=status.HTTP_200_OK)
 
 
+class CreateInvestmentAPIView(APIView):
+    authentication_classes = (JSONWebTokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request):
+        user_id = request.user.id
+        user = User.objects.get(id=user_id)
+        base_symbol = request.query_params['base']
+        target_symbol = request.query_params['target']
+
+        base_equipment = get_object_or_404(Equipment, symbol=base_symbol)
+        target_equipment = get_object_or_404(Equipment, symbol=target_symbol)
+        base_amount = request.query_params['base_amount']
+        target_amount = request.query_params['target_amount']
+        ManualInvestment(base_equipment=base_equipment, target_equipment=target_equipment,
+                         base_amount=base_amount, target_amount=target_amount, made_by=user).save()
+
+        return Response(status=status.HTTP_200_OK)
+
+
+class DeleteInvestmentAPIView(APIView):
+    authentication_classes = (JSONWebTokenAuthentication,)
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request):
+        user_id = request.user.id
+        user = User.objects.get(id=user_id)
+        investment_id = request.query_params['id']
+        if not ManualInvestment.objects.filter(id=investment_id).exists():
+            Response({
+                'message': 'User does not have investment with id: ' + str(investment_id)},
+                status=status.HTTP_404_NOT_FOUND)
+
+        ManualInvestment.objects.filter(id=investment_id).delete()
+        return Response(status=status.HTTP_200_OK)
+
```

Comment:
1 - use DELETE method for delete view (def delete() ...)
2 - receive parameters with request.data not query_params. 
3 - u can also merge DeleteInvestmentAPIView and CreateInvestmentAPIView into one class CreateDeleteInvestmentAPIView